test/hotspot/jtreg/testlibrary_tests/template_framework/tests/TestTemplate.java

```diff
@@ -1219,37 +1563,39 @@ class X_1 {
     public static void testDataNames3() {
         var hook1 = new Hook("Hook1");
 
-        var template0 = Template.make("type", "mutability", (DataName.Type type, DataName.Mutability mutability) -> body(
+        var template0 = Template.make("type", "mutability", (DataName.Type type, DataName.Mutability mutability) -> scope(
             "  #mutability: [",
-            dataNames(mutability).exactOf(myInt).hasAny(),
+            dataNames(mutability).exactOf(myInt).hasAny(h -> scope(h)),
             ", ",
-            dataNames(mutability).exactOf(myInt).count(),
+            dataNames(mutability).exactOf(myInt).count(c -> scope(c)),
```

Comment:
There are many more occurrences further down. Feel free to fix them but I guess it's not that important.
```suggestion
            dataNames(mutability).exactOf(myInt).hasAny(Template::scope),
", ",
            dataNames(mutability).exactOf(myInt).count(Template::scope),
```